nav2_navfn_planner/src/navfn.cpp

```diff
@@ -295,18 +295,7 @@ NavFn::calcNavFnDijkstra(bool atStart)
   setupNavFn(true);
 
   // calculate the nav fn and path
-  propNavFnDijkstra(std::max(nx * ny / 20, nx + ny), atStart);
-
-  // path
-  int len = calcPath(nx * ny / 2);
-
-  if (len > 0) {  // found plan
```

Comment:
You're missing if the path length is 0, the false that prop returns is only if it exceeded the cycle counter